styles.css

```diff
@@ -1,35 +1,469 @@
 
+
+header {
+    font-family: 'Roboto', sans-serif;
+    color: white;
+
+}
+
+h1 {
+    font-family: 'Merriweather', serif;
+    font-size: 40px;
+    text-align: center;
+
+}
+
+.hello {
+    font-family: 'Merriweather', serif;
+    text-align: center;
+    font-size: 20px;
+    color: var(--purple-text);
+
+}
+
+.products h3 {
+    font-size: 16px;
+}
+
+
+.hello-contact {
+    color: var(--main-text);
+    font-family: 'Roboto', sans-serif;
+    font-size: 15px;
+}
+
+
+h2 {
+    font-family: 'Merriweather', serif;
+}
+
+.text-rollingstone, .text-variety, .text-worth p {
+    font-family: 'Roboto', sans-serif;
+    font-size: 15px;
+}
+
+.products h3 {
+    font-family: 'Merriweather', serif;
+    color: var(--purple-text);
+
+}
+
+.products p {
+    font-family: 'Roboto', sans-serif;
+    font-size: 12px;
+    color: var(--service-text);
+
+
+}
+
+.contacttext {
+    color: white;
+}
+
+.contacttext p {
+    font-family: 'Roboto', sans-serif;
+}
+
+.form {
+    color: white;
+    font-family: 'Roboto', sans-serif;
+    font-weight: bold;
+    font-size: 10px;
+}
+
+a:-webkit-any-link {
+    font-family: 'Roboto', sans-serif;
+    color: mediumpurple;
+    cursor: pointer;
+    text-decoration: none;
+    text-transform: uppercase;
+}
+
+
+header a:visited {
+    color: white;
+    text-transform: none;
+    font-family: 'Roboto', sans-serif;
+}
+
+header a:link {
+    color: white;
+    text-transform: none;
+    font-family: 'Roboto', sans-serif;
+}
+
+#capital {
+    text-transform: uppercase;
+}
+
+main a:link {
+    color: var(--purple-text);
+    font-family: 'Roboto', sans-serif;
+    text-transform: uppercase;
+}
+
+main a:visited {
+    color: var(--purple-text);
+    font-family: 'Roboto', sans-serif;
+    text-transform: uppercase;
+}
+
 /* ------------------------------- 3. Layout --------------------------------------- */
+.button {
+    background-color: var(--buttons);
+    color: #fff;
+    border-radius: 30px;
+    padding: 20px;
+    margin: 10px;
+    font-size: 12px;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+    box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.24);
+    border-color: rgba(0, 0, 0, 0.0);
+}
+
+.submitbutton {
+    background-color: var(--buttons);
+    color: #fff;
+    border-radius: 30px;
+    padding: 20px;
+    margin: 10px;
+    font-size: 12px;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+    box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.24);
+    border-color: rgba(0, 0, 0, 0.0);
+
+}
+
+::placeholder {
+    font-family: 'Roboto', sans-serif;
+    padding: 0 0 0 5px;
+    font-weight: bold;
+    font-size: 10px;
+}
+
+.image-wrapper {
+    flex-basis: 800px;
+    box-shadow: 0 25px 22px 0px rgba(0, 0, 0, 0.21);
+    flex-grow: 0;
+    flex-shrink: 1;
+}
+
+img {
+    width: 100%;
+    flex-grow: 0;
+    flex-shrink: 1;
+}
 
 /* --------------- 3.1 Reusables ----------------------- */
+.outer-container {
+    border: solid 1px rgba(0, 0, 0, 0.14);
```

Comment:
Heb je die border met een reden toegepast? Ik zie als ik de pagina start heel veel borders namelijk. Ik de voorbeeld afbeeldingen staan deze niet namelijk

Reply:
Bedankt! 
Omdat ik het originele voorbeeld niet meer geopend kreeg, kon ik niet zien of er oorspronkelijk borders tussen de pagina's zaten of niet, dus ik heb deze 'test-borders' maar open laten staan. Zonder vond ik het er zo gek uit zien haha